### PR TITLE
Shortest Distance Between Words #1685

### PR DESCRIPTION
**Pull request by ErzaTitania-2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

(Write your answer here.)

### Checklist

- [ y] I've read the contribution guidelines.
- [ y] I've checked the issue list before deciding what to submit.
- [ y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Shortest Distance Between Words #1685 as assigned by @shraddhavp

### CODE REVIEW

**Comment by ErzaTitania-2001** [comment]
@iamrajiv.

File: Java/cp/Shortest_Dist.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ */
+import java.util.*;
+class Shortest_Dist {
+    static List < String > arr;
+    public static void main(String args[]) {
+        int t = 0;
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the list of words " + "\n" + "Words = ");
+        arr = new ArrayList < > ();
```
Comment (vatsalkesarwani12) [inline Java/cp/Shortest_Dist.java]: remove spaces between <>

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

File: Java/cp/Shortest_Dist.java (hunk 1 above)
Comment (ErzaTitania-2001) [reply Java/cp/Shortest_Dist.java]: @plazzy99 done

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

File: Java/cp/Shortest_Dist.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            if (sc.hasNextInt()) {
+                t = sc.nextInt();
+                break;
+            }
+
+        }
+
+        List < String > wd = new ArrayList(arr);
+        while (t > 0) {
+            String wd1 = sc.next();
```
Comment (atarax665) [inline Java/cp/Shortest_Dist.java:27]: Prompt user to enter the words between which the distance is to be determined.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Add comments wherever needed, it will help people to understand the approach used.

File: Java/cp/Shortest_Dist.java (hunk 2 above)
Comment (ErzaTitania-2001) [reply Java/cp/Shortest_Dist.java:27]: @atarax665 done.

**Review by iamrajiv — APPROVED** [review]

**Comment by ErzaTitania-2001** [comment]
@iamrajiv  what is DeepSource error?

**Comment by iamrajiv** [comment]
@ErzaTitania-2001 DeepSource is a code analysis tool. Do not worry about that. It is yet to be set for repository properly.

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**